### ISSUE TITLE
question about Hessian-vector products 

### ISSUE THREAD

**Issue by shawnkx** [dragen1860/MAML-Pytorch#18]
In the original paper, the authors claimed that MAML needs second gradient and Hessian-vector products. Could you explain how do you implement this or Pytorch just do this automatically? Thanks!

**Comment by dragen1860** [comment]
@shawnkx Does the official MAML git repo. implement hessian vector product ? I forgot already.

**Comment by cLjx** [comment]
The results in the original paper claim K-shot, which comes to a doubt that the K means the number on the stage of meta-training of meta-testing (fine-tuning) with regard to the different shot number in the code.

**Comment by seanie12** [comment]
@dragen1860 
In the official MAML you can choose whether to use second order deriative as follows
https://github.com/cbfinn/maml/blob/master/main.py#L60 
https://github.com/cbfinn/maml/blob/master/maml.py#L92
